src/Symfony/Component/HttpClient/DataCollector/HttpClientDataCollector.php

```diff
@@ -98,6 +98,7 @@ private function collectOnClient(TraceableHttpClient $client): array
         $errorCount = 0;
         $baseInfo = [
             'response_headers' => 1,
+            'retry_count' => 1,
```

Comment:
This looks confusing to me. `retry_count: 1` would tell me that this request was retried once, and so that 2 HTTP queries were performed.

Reply:
hmm.. You're right `retry_count: 1` means request has been retried once.

here, `1` is a "never used dummy" value.
The `$baseInfo` is used at line 161 to extract "debug infos" with `array_diff_key($info, $baseInfo)`.